addons/website_event_sale/static/tests/tours/helpers/WebsiteEventSaleTourMethods.js

```diff
@@ -0,0 +1,70 @@
+odoo.define('website_event_sale.tour.WebsiteEventSaleTourMethods', function (require) {
+    'use strict';
+
+    class EventTools {
```

Comment:
I don't think we need a class to generate the steps here (or with a constructor to take in the 'tour object' parameters, then you could have a "generate_tour_steps" *instance* method that uses the object values, but it is an overkill here).
 
Even without a class, you could have that method that returns the aggregation of the three other methods, so that you wouldn't have to duplicate the code for these calls either, especially with the comments describing the case (can maybe be removed from commit message?). 

So, something like 
```js
const get_pricelist_checks_steps = function (pricelistName, event, price, price_before_discount=False) {
   // calling other methods and returning array of steps
}
return {
    get_pricelist_checks_steps,
    // other methods needed for WebsiteEventBoothSaleTourMethods
 };
```
and in website_event_sale_pricelists.js:

```js
const { get_pricelist_checks_steps } = require('website_event_sale.tour.WebsiteEventSaleTourMethods');

...
    /*
    Test Case 1:
    1) Change pricelist to "EUR With Discount Included"
    2) Verify price of the event -> 90.00
    3) Verify price in the cart -> 90.00
    */
    ...get_pricelist_checks_steps("EUR With Discount Included", "Pycon", "90.00")
    // when with discount ...
    ...get_pricelist_checks_steps(
        "EUR Without Discount Include", "Pycon", "90.00", price_before_discount="100.00"
    )
```

Reply:
Ok I adapted it